### DISCUSSION TITLE
Running pac commands through an Azure Function returns error

### DISCUSSION THREAD

**Discussion by Michael-Kim0717** [microsoft/powerplatform-build-tools#1052] [Q&A]
Hi, I am trying to run some pac copilot commands via an Azure Function app and am receiving the following error:

`Error: Unable to open a web page using xdg-open, gnome-open, kfmclient or wslview tools. See inner exception for details. Possible causes for this error are: tools are not installed or they cannot open a URL. Make sure you can open a web page by invoking from a terminal: xdg-open https://www.bing.com`

I validated this locally, but it seems to only be an issue when run through the Function app itself.

**Comment by mannybellas** [comment]
Interesting!  looks like `pac copilot` relies on Windows Subsystem for Linux (WSL).
